policy.go

```diff
@@ -23,8 +23,6 @@ import (
 
 const (
 	policyType = "application/vnd.ibm.kms.policy+json"
-
-	RotationPolicy = "rotation"
```

Comment:
Instead of moving RotationPolicy to instances.go, we can move dual-auth-delete to policy.go so that all the key policy related constants stay there